programs/bpf_loader/src/serialization.rs

```diff
@@ -437,24 +437,26 @@ fn serialize_parameters_aligned(
                 size += size_of::<u8>() // is_signer
                 + size_of::<u8>() // is_writable
                 + size_of::<u8>() // executable
-                + size_of::<u32>() // original_data_len
+                + 4 // padding  of 4 bytes after executable
+                + size_of::<u64>() // original_data_len
                 + size_of::<Pubkey>()  // key
                 + size_of::<Pubkey>() // owner
                 + size_of::<u64>()  // lamports
                 + size_of::<u64>()  // data len
-                + MAX_PERMITTED_DATA_INCREASE
                 + size_of::<u64>(); // rent epoch
                 if copy_account_data {
-                    size += data_len + (data_len as *const u8).align_offset(BPF_ALIGN_OF_U128);
+                    let align_offset = (data_len as *const u8).align_offset(BPF_ALIGN_OF_U128);
+                    size += data_len + align_offset;
+                    size += MAX_PERMITTED_DATA_INCREASE + align_offset;
```

Comment:
align_offset is added twice here

Reply:
Check here while account serialization align offset is added for data_len and max_permitted_data_increase [
](https://github.com/anza-xyz/agave/blob/f36a62dce8bbd9ee41c93626e86872241df31119/programs/bpf_loader/src/serialization.rs#L110)

Reply:
it does not need to be added twice

Reply:
Ok will test without it